scripts/alert_emails/readme.md

```diff
@@ -1,12 +1,18 @@
 # Email Alerting
 
-The general process for email alerting is as follows
+CAN sends email alerts to subscribed users notifying them when states/counties change risk levels. These are automated via a github workflow (see .github/workflows/send-alert-emails.yml).
```

Comment:
I was fixing this doc to not mention campaign monitor and just took an overall pass at cleaning it up a bit.